explorer/testdata/operators/add_builtin.carbon

```diff
@@ -11,9 +11,6 @@
 
 package ExplorerTest api;
 
-// TODO: T:! Add
-fn DoAdd[T:! AddWith(.Self) where .Result == .Self](x: T, y: T) -> T { return x + y; }
-
```

Comment:
I think this is an unrelated change due to needing to merge up something...

Reply:
Fixed